Description: Fix include path for phonon4qt5
Author: https://aur.archlinux.org/packages/yarock-qt5/
Last-Update: 2019-01-29
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/

Index: Yarock_1.3.1_Sources/src/core/player/phonon/CMakeLists.txt
===================================================================
--- Yarock_1.3.1_Sources.orig/src/core/player/phonon/CMakeLists.txt	2019-02-01 21:05:13.000000000 +0000
+++ Yarock_1.3.1_Sources/src/core/player/phonon/CMakeLists.txt	2019-02-01 21:05:20.000000000 +0000
@@ -34,7 +34,7 @@
 
 
 if (ENABLE_QT5)
-    INCLUDE_DIRECTORIES( ${PHONON_INCLUDE_DIR}/phonon4qt5 )
+    INCLUDE_DIRECTORIES( /usr/include/phonon4qt5 )
 else (ENABLE_QT5)
     INCLUDE_DIRECTORIES( ${PHONON_INCLUDES} )
 endif (ENABLE_QT5)  
